code/game/objects/items/melee/energy.dm

```diff
@@ -192,6 +192,16 @@
 	else
 		return ..()
 
+/obj/item/melee/transforming/energy/sword/bee  //yeah its fucking stupid but I wanted a yellow esword which is weaker than what we have
+	name = "Bee Sword"

```

Comment:
the stinger

Reply:
Look i hate the fact that i had to code this already please dont make me revisit that file

Reply:
i guess i ll have to make it cool myself duhhhhhhhh also module filling isnt coding

Reply:
if there are slashes and lots of lines it’s coding idiot read a book